fts/Dockerfile

```diff
@@ -11,12 +11,12 @@ RUN yum install -y \
     mysql \
     multitail \
     gfal2-plugin* \ 
-    fts-libs-3.14.0-r2404100902git47e773b.el7.cern.x86_64 \
-    fts-server-3.14.0-r2404100902git47e773b.el7.cern.x86_64 \
-    fts-rest-server-3.14.0-r2404100848gitb7554d5.el7.cern.noarch \ 
+    fts-libs-3.13.0-r2405271344gitc6d02cc.el7.cern.x86_64 \
```

Comment:
It's a more fundamental discussion we need to decide on. But unless dependabot also auto-suggests updates here, I feel like this will be something we will always forget to update.

Reply:
We can schedule a periodic task to work on this, e.g. even a GH action that checks the repo and sees if there's a later version, raising it as a warning. I tried finding some Dependabot setting to do this, but I'm not sure if it's feasible.

In this context, I think it's better to have pinned dependencies and end up with possibly outdated dependencies, rather than having these sudden test failures